docs/manual/kinds/keysight_ixia-c-one.md

```diff
@@ -0,0 +1,132 @@
+# Keysight IXIA-C One
+
+Keysight ixia-c-one is a single-container distribution of [ixia-c][ixia-c], which in turn is Keysight's reference implementation of [Open Traffic Generator API][otg].
+
+!!!info "What is IXIA-C?"
+    Ixia-c is a modern, powerful and API-driven traffic generator designed to cater to the needs of hyperscalers, network hardware vendors and hobbyists alike.
+
+    It is available **for free** and distributed / deployed as a multi-container application consisting of a [controller](https://hub.docker.com/r/ixiacom/ixia-c-controller), a [traffic-engine](https://hub.docker.com/r/ixiacom/ixia-c-traffic-engine) and an [app-usage-reporter](https://hub.docker.com/r/ixiacom/ixia-c-app-usage-reporter).
+
+The corresponding node in containerlab is identified with `keysight_ixia-c-one` kind in the [topology file](../topo-def-file.md). Upon boot up, it comes up with:
+
+- management interface `eth0` configured with IPv4/6 addresses as assigned by container runtime
+- hostname assigned to the node name
+- HTTPS service enabled on port 443 (for client SDK to push configuration and fetch metrics)
+
+## Managing ixia-c-one nodes
+
+ixia-c-one is a "docker in docker" container hosting two kinds of [ixia-c][ixia-c] containers internally:
+
+- A set of containers acting as API endpoint and managing configuration across multiple test ports
+- A set of containers bound to network interface (created by containerlab), treating it as a test port (i.e. for generating or processing traffic, emulating protocols, etc.)
+
```

Comment:
can you please give me a description which container does what? I will tune the docs to reflect this in more details:

1. ixia-c-port-cp-eth1
2. ixia-c-port-dp-eth1
3. ixia-c-gnmi-server
4. ixia-c-controller

Reply:
@hellt I kept it short on purpose as there's a difference in free vs commercial version of ixia-c-one.

- `ixia-c-controller` - acts as an API endpoint (HTTPS) and manages configuration across multiple test ports (available in both versions)
- `ixia-c-port-dp-eth1` - used for generating and processing data-plane traffic (available in both versions)
- `ixia-c-port-cp-eth1` - used for generating and processing control-plane traffic and emulating protocols (available only in commercial version)
- `ixia-c-gnmi-server` - acts as a gNMI endpoint and internally talks to ixia-c-controller over HTTPS (included in neither of the versions, but we plan to include it in future if there's a need)

Reply:
nice. Updated the image and now have a clearer picture.